src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiFactory.java

```diff
@@ -20,6 +22,17 @@ protected boolean isMatch(@Nullable String serverUrl) {
     @Override
     protected BitbucketApi create(@Nullable String serverUrl, @Nullable StandardUsernamePasswordCredentials credentials,
                                   @NonNull String owner, @CheckForNull String repository) {
-        return new BitbucketCloudApiClient(owner, repository, credentials);
+        AbstractBitbucketEndpoint endpoint = BitbucketEndpointConfiguration.get().findEndpoint(BitbucketCloudEndpoint.SERVER_URL);
+        boolean enableCache = false;
+        int teamCacheDuration = 0;
+        int repositoriesCacheDuration = 0;
+        if (endpoint != null && endpoint instanceof BitbucketCloudEndpoint) {
+            enableCache = ((BitbucketCloudEndpoint) endpoint).isEnableCache();
+            teamCacheDuration = ((BitbucketCloudEndpoint) endpoint).getTeamCacheDuration();
+            repositoriesCacheDuration = ((BitbucketCloudEndpoint) endpoint).getRepositoriesCacheDuration();
+        }
+        return new BitbucketCloudApiClient(
+                enableCache,teamCacheDuration,repositoriesCacheDuration,
```

Comment:
Could you follow existent coding style here please? (space after `,`)

Reply:
done